### ISSUE TITLE
Pata nhi

### ISSUE THREAD

**Issue by ScizorScara** [ScizorScara/Street-light-L#2]
`<!DOCTYPE html>
<html lang="en">
<head>
    
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.0/jquery.min.js"></script>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Task</title>
</head>
<body>
    <h1>Registration Form for <strong>Maxval</strong> Technologies</h1>
    <section class="taskform">
        <p>
            Sign Up
        </p>
        <form>
        Name <input id="name_frame" type="text" name="form_name" >
        <div id="nameError" style="background-color: red;">

        </div><br><br>
        E-mail <input id="Email" type="email" name="form_email" required><br><br>
        Contact <input id="number" type="tel" name="form_number" pattern="[0-9]{10}" required><br><br>
        <input type="radio" id="gender" name="form_gender" value="Male">
        <label for="Male">Male</label><br>
        <input type="radio" id="gender" name="form_gender" value="Female">
        <label for="Female">Female</label><br>
        <input type="radio" id="gender" name="form_gender" value="unassigned">
        <label for="unassigned">unassigned</label><br><br>
        <button id="submit" type="submit">Submit</button>
        <button id="remove" type="reset">Delete</button>
        </form>
    </section>
    <section>
        <table class="tbl">
            <tr class="ttr">
                <th class="taskyn">Name</th>
                <th class="taskye">Email</th>
                <th class="taskm">Mobile Number</th>
                <th>Gender</th>
            </tr>

            

        </table>
    </section>
  
  
  
  
    <link rel="stylesheet" href="styles.css">
    <script src="final.js"></script>

</body>
</html>`
